src/components/text/index.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames'
+import { TextProps } from './types'
+import s from './text.module.css'
+
+const Text: React.FC<TextProps> = ({
+  asElement = 'p',
+  size = 300,
```

Comment:
[issue?] I see you're setting `300` as default size. I understand you're pro larger text in the UI, totally get it. But in case in the future the design system may decide to bump up the size of the text scale for body, then all the places where `300` is used by default may need to be revisited? I am not sure is an issue per se, since I expect in any case the developers will follow the specifications of the designs in Figma, but I wanted to raise anyway the possibility of a possible side-effect down the line.

Reply:
For now, setting the default value to `300` is most efficient for us since it's used significantly more than the other size values. I don't currently see this as an issue since this is our first pass at this component and we can't build for every case right away, but I do think the situation you mention is straightforward to address if/when we come across it!